pkg/minishift/registration/utils.go

```diff
@@ -26,12 +26,18 @@ import (
 )
 
 type RegistrationParameters struct {
-	Username string
-	Password string
+	Username           string
+	Password           string
+	SkipRegistration   bool
+	SkipUnRegistration bool
 }
 
 // Register host VM
 func RegisterHostVM(host *host.Host, param *RegistrationParameters) error {
+	if param.SkipRegistration {
+		log.Debug("User choosed to skip registration")
```

Comment:
I second @praveenkumar. This is imo classic info which should be logged through the logger. In fact we should refrain as much as possible from logging straight to the console from within anything in _pkg_. Otherwise it will be hard to control. 

@praveenkumar I suggest to formulate this a bit different though. "Skipping registration due to enabled --skip-registration flag". (or something like this). BTW, "choosed" does not exist ;-)